### DISCUSSION TITLE
Upgrading workflows to ubuntu 24.04

### DISCUSSION THREAD

**Discussion by CodeDoctorDE** [LinwoodDev/Butterfly#739] [Announcements]
Hello,
based on the github actions timeline (https://github.com/actions/runner-images/issues/10636) the butterfly repository will start building binaries with the ubuntu 24.04 with these libraries.
This could lead to incompatibilities with older distributions.
Because of this, there is always an alternative portable version for older distributions. These were previously build with ubuntu 20.04. Because the regular support on this distribution ends in 6 months (https://endoflife.date/ubuntu) this version will be built with ubuntu 22.04.

TLDR:
The ubuntu 22.04 binaries will be moved to alternative portable.
The previous alternative ubuntu 20.04 won't be accessible. You could build it yourself if you want.
The new ubuntu 24.04 will be the default.

The migration starts with Butterfly 2.2.0-rc.1.
